ajax/libs/jshashtable.js/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "jshashtable.js",
+  "version": "3.0",
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Tim Down",
+    "email": "tim@timdown.co.uk"
+  },
+  "homepage": "https://www.timdown.co.uk/jshashtable/index.html",
+  "filename": "hashtable.min.js",
+  "description": "A standalone JavaScript implementation of hash table with arbitrary keys",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/timdown/jshashtable.git"
+  },
+  "keywords": [
+    "javascript",
+    "hash",
+    "hastable",
+    "hashset"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/timdown/jshashtable.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "hashset.js",
+          "hashtable.js"
```

Comment:
Let's use `+(hashset| hashtable)*.+(js|css|map)`
we can use one line of pattern to get all the files we need. (via  [node-glob](https://github.com/isaacs/node-glob#glob-primer))
What do you think?

Thank you.